cmd/kubeadm/app/cmd/certs.go

```diff
@@ -40,6 +35,10 @@ import (
 	kubeconfigphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/kubeconfig"
 	configutil "k8s.io/kubernetes/cmd/kubeadm/app/util/config"
 	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
+
+	"github.com/lithammer/dedent"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
```

Comment:
the imports are ordered like that intentionally

stdlib
external
external k8s.io
kubeadm

Reply:
I see, thanks